mitodl/mit-open
# codespace fix for nextjs branch

### What are the relevant tickets?

Closes https://github.com/mitodl/hq/issues/5604

### Description (What does it do?)
This PR gets the nextjs branch to work in codespaces.


### How can this be tested?
1. create a codespace on this branch (you may need to launch this in a 4 core container )
2. Once you see 3 ports show up in the ports tab at the bottom, right click port 8063 and set it to public
3. visit port 8062 in the browser (it may not immediately load which means its still building but it eventually will show up
4. poke around the site and see that its all there. the search page should populate with resources

### Additional Context
There were a few configuration related to nextjs that had to go in. opensearch also was not initializing presumably due to changes in some indexing operations (the service fails in codespace environment). setting a restart policy for opensearch fixed this.
